Makefile

```diff
@@ -1,5 +1,5 @@
 SKIP_SQUASH?=0
-VERSIONS="1"
+VERSIONS="1 slave-base slave-maven slave-nodejs"
```

Comment:
Cool ... IGTM

On Tue, Jul 12, 2016 at 6:49 PM, Ben Parees <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/openshift/jenkins/pull/122#discussion_r70536771>:
>
> > @@ -1,5 +1,5 @@
> >  SKIP_SQUASH?=0
> > -VERSIONS="1"
> > +VERSIONS="1 slave-base slave-maven slave-nodejs"
>
> done
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/jenkins/pull/122/files/492075ba4f7d834da15e1d905aa74c5969c36628#r70536771>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADbadDGR9KD6gcyHMHYGxoZRTTw4lRtcks5qVBnrgaJpZM4JIXVg>
> .
>